cpp/FileTransferAgent.cpp

```diff
@@ -616,12 +617,14 @@ void Snowflake::Client::FileTransferAgent::compressSourceFile(
   stagingFile += m_stmtPutGet->UTF8ToPlatformString(fileMetadata->destFileName);
   std::string srcFileNamePlatform = m_stmtPutGet->UTF8ToPlatformString(fileMetadata->srcFileName);
 
-  FILE *sourceFile = fopen(srcFileNamePlatform.c_str(), "r");
+  FILE *sourceFile = NULL;
+  sb_fopen(&sourceFile, fileMetadata->srcFileName.c_str(), "r");
```

Comment:
@sfc-gh-ext-simba-jz  merging issue that breaks put on GCP with unicode characters.
Please double check your changes to make sure no logic changes beside replacement.

Reply:
fixed. It's a merge mistake.